cmd/installer/pkg/install.go

```diff
@@ -29,12 +30,22 @@ func Install(p runtime.Platform, config runtime.Configurator, sequence runtime.S
 	cmdline := kernel.NewCmdline("")
 	cmdline.Append("initrd", filepath.Join("/", "default", constants.InitramfsAsset))
 	cmdline.Append(constants.KernelParamPlatform, p.Name())
-	cmdline.Append(constants.KernelParamConfig, opts.ConfigSource)
 
 	if err = cmdline.AppendAll(p.KernelArgs().Strings()); err != nil {
 		return err
 	}
 
+	if option := cmdline.Get(constants.KernelParamConfig).First(); option == nil {
```

Comment:
This will always be true since we create an empty command line with `kernel.NewCmdline("")`.

Reply:
Negative, see L34 -- 
```
if err = cmdline.AppendAll(p.KernelArgs().Strings()); err != nil {
		return err
	}
```
This should only be the case when a platform does not provide a location / configuration. It's fine if the platform provides `talos.config=none` since the platform is being explicit.